third_party/mbedtls/repo/library/ctr_drbg.c

```diff
@@ -178,7 +178,7 @@ static int block_cipher_df( unsigned char *output,
     buf_len = MBEDTLS_CTR_DRBG_BLOCKSIZE + 8 + data_len + 1;
 
     for( i = 0; i < MBEDTLS_CTR_DRBG_KEYSIZE; i++ )
-        key[i] = i;
+        key[i] = (unsigned char)i;
```

Comment:
Looking at the [`README.google`](https://github.com/openthread/openthread/blob/master/third_party/mbedtls/README.google) file, it looks like https://github.com/ARMmbed/mbedtls